encryptionkeyrotation/README.md

```diff
@@ -77,11 +77,16 @@ message EncryptionKeyRotateRequest {
   // This field is OPTIONAL
   string encryption_key = 3 [(csi.v1.csi_secret) = true];
 
+  // Volume capability describing how the CO intends to use this volume.
+  // This allows SP to determine if volume is being used as a block
+  // device or mounted file system. This is OPTIONAL.
+  csi.v1.VolumeCapability volume_capability = 4;
```

Comment:
Are the important attributes for this not available through the `volume_id` already? Or is there not an other way to identify/detect this while performing the `EncryptionKeyRotateRequest`?

Reply:
Marking it as resolved wrt offline discussions.